lldb/source/Plugins/ScriptInterpreter/None/ScriptInterpreterNone.cpp

```diff
@@ -26,17 +26,19 @@ ScriptInterpreterNone::ScriptInterpreterNone(Debugger &debugger)
 
 ScriptInterpreterNone::~ScriptInterpreterNone() = default;
 
+static const char *no_interpreter_err_msg =
+    "There is no embedded script interpreter. Check that LLDB was built with a "
+    "scripting language enabled.\n";
```

Comment:
[begin bikeshedding] 

How about:

> Embedded script interpreter unavailable: LLDB was built without scripting language support.

"Check" makes it sound like that's something the user has control over, which is only really the case for us developers. 


[end bikeshedding]

Reply:
Right, this also doesn't imply that not having scripting support is necessarily bad. It's just a choice made at some point.